macros/sfdc_formula_view_sql.sql

```diff
@@ -1,7 +1,7 @@
 {%- macro sfdc_formula_view_sql(join_to_table, source_name = 'salesforce', reserved_table_name=join_to_table, inclusion_fields=none, not_null_value=true) -%}
 
     --Generate the key value pair from the formula field table with the below macro.
-    {%- set key_val = salesforce_formula_utils.sfdc_get_formula_column_values(source(source_name, 'fivetran_formula'), 'field', 'view_sql', join_to_table, inclusion_fields, not_null_value) -%}
+    {%- set key_val = salesforce_formula_utils.sfdc_get_formula_column_values(source(source_name, 'fivetran_formula'), 'field', 'view_sql', join_to_table, not_null_value) -%}
```

Comment:
I believe we need to add a similar `if k in added_inclusion_fields` condition within this model. Specifically at this point:
https://github.com/fivetran/dbt_salesforce_formula_utils/blob/8c1f75707e4c6c916b9bd6c4dd5834dc1c272745/macros/sfdc_formula_view_sql.sql#L10

Otherwise, non included `view_sql` formulas can still be compiled in the output query. This then causes the job to fail since we are filtering these records our in the `sfdc_formula_view_fields` macro, but not here.

Additionally, I think this macro is a perfect example of why we should try and consider a way to optimize the code. To make this dynamic for both inclusion and normal runs, we will need to have the large conditional block twice. If there is a way we could optimize this and clean up the conditions then that would be ideal.